installing/install_config/customizations.adoc

```diff
@@ -29,46 +29,55 @@ Custom Resource Definitions (CRDs) that are deployed on your cluster.
 |Configure the Kubernetes API server.
 
 |Authentication
-|You can configure the identity provider for your cluster.
+|You can configure the
+xref:../../authentication/understanding-identity-provider.adoc#understanding-identity-provider[identity provider]
+for your cluster.
 
 |Build
 |Configure build settings.
 
 |ClusterVersion
-|Configure the desired version of the cluster and how updates are handled.
+|In {product-title} {product-version}, you must not customize the ClusterVersion
+resource for production clusters. Instead, follow the process to
+xref:../../updating/updating-cluster.adoc#updating-cluster[update a cluster].
 
 |Console
 |Configure the web console.
 
 |DNS
-|Configure DNS.
+|To configure DNS by using the Network Operator, you must
+xref:../../installing/installing_aws/installing-aws-network-customizations.adoc#installing-aws-network-customizations[customize networking during installation].
 
-|Feature
-|Configure feature gates.
+|FeatureGate
+|You can
+xref:../../nodes/clusters/nodes-cluster-enabling-features.adoc#nodes-cluster-enabling[enable a FeatureGate]
+so that you can use Tech Preview features.
 
 |Image
-|Configure image registries.
+|Configure the
+xref:../../openshift_images/configuring-samples-operator.adoc#configuring-samples-operator[samples Operator]
+to start using images.
 
 |Infrastructure
-|In {product-title} {product-version}, you cannot customize the infrastructure resource.
+|In {product-title} {product-version}, you cannot customize the Infrastructure resource.
 
 |Ingress
-|Configure Ingress.
+|To enable external access to {product-title} cluster services, configure the
+xref:../../networking/ingress/configuring-ingress-operator.adoc#configuring-ingress[Ingress Operator].
 
 |Network
```

Comment:
@jboxman, what's the difference between the DNS type and the Network type? Do you configure both during installation, or does one have a separate link?

Reply:
To be honest, I don't know anything at all about DNS. The Cluster Network Operator deals entirely with the cluster network, so IP address ranges, Pod isolation, NetworkPolicy, and so forth. Maybe @bmcelvee knows? I think DNS is handled by the edge network team?